int/hwsc-user-svc/hwsc-user-svc.js

```diff
@@ -1,18 +1,15 @@
 const grpc = require('grpc');
 const protoLoader = require('@grpc/proto-loader');
 
-const HWSC_USER_PROTO_PATH = '../lib/';
-const HWSC_USER_SVC_PROTO_PATH = `${__dirname}/proto/`;
+const PROTO_PATH = `${__dirname}/../..`;
 
 const options = {
-  includeDirs: [
-    HWSC_USER_PROTO_PATH,
-    HWSC_USER_SVC_PROTO_PATH,
-  ],
+  includeDirs: [PROTO_PATH],
 };
 
 // load the .proto files
-const packageDefinition = protoLoader.loadSync('hwsc-user-svc.proto', options);
+const packageDefinition = protoLoader
+  .loadSync('int/hwsc-user-svc/user/hwsc-user-svc.proto', options);
```

Comment:
With `path` set as `github.com/hwsc-org/hwsc-api-blocks', we need to find the proto at `'int/hwsc-user-svc/user/hwsc-user-svc.proto'`.

Any proto that has a path such as `import lib/user.proto` would be found within the `path`